android/lib/src/main/java/com/ern/api/impl/navigation/ElectrodeBaseActivity.java

```diff
@@ -85,6 +102,16 @@ public boolean onKeyUp(int keyCode, KeyEvent event) {
         return super.onKeyUp(keyCode, event);
     }
 
+    private void setupNavBar() {
```

Comment:
Just a side-node/question: Do you want to move private helper methods to the bottom of the class (below public members?). It would follow the default Eclipse/IntelliJ layout